testing/trino-testing-services/src/main/java/io/trino/testing/FlakyTest.java

```diff
@@ -28,7 +30,9 @@
 @Deprecated
 @Retention(RUNTIME)
 @Target(METHOD)
-public @interface Flaky
+@TestTemplate
+@ExtendWith(FlakyTestRetryExtension.class)
+public @interface FlakyTest
```

Comment:
From the commit message:

> This is both a TestNG and a JUnit5 annotation, and in the latter it's used as a standalone annotation. It looks weird to have a method annotated with just `@Flaky`; it's a test method, so it's more readable when it's annotated with `@FlakyTest`.

I know it looks weird to have `@Test @FlakyTest`, but it looks even weirder in JUnit5 when it's just `@Flaky`. And TestNG will go away eventually, so that weirdness will disappear.